client/src/app/auth/forgot-password/page.tsx

```diff
@@ -0,0 +1,135 @@
+"use client";
+import { useState, useEffect } from "react";
+import useFetch from "@/src/hooks/useFetch";
+import { AuthService } from "@/src/services/AuthService";
+import Link from "next/link";
+import Navbar from "@/src/components/Navbar/Navbar";
+import Footer from "@/src/components/Footer/Footer";
+import useThemeStore from "@/src/store/themeStore";
+import useAuthStore from "@/src/store/authStore";
+import Image from "next/image";
+import ForgotPasswordSvg from "@/public/assets/Login.svg";
+
+const ForgotPassword = () => {
+  const theme = useThemeStore((state) => state.theme);
+   const authToken = useAuthStore((state) => state.authToken);
+  
+  const [email, setEmail] = useState("");
+  const [errorMessage, setErrorMessage] = useState("");
+  const [successMessage, setSuccessMessage] = useState("");
+
+  const [{ isLoading, data, isError, error }, forgotPasswordAPI] = useFetch(null);
```

Comment:
Follow the same object as we are using everywhere for custom hook. Why error is present as its already a isError is present.